src/node_util.cc

```diff
@@ -174,6 +176,15 @@ void PromiseReject(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(ret.FromMaybe(false));
 }
 
+void SafeGetenvForJs(const FunctionCallbackInfo<Value>& args) {
+  CHECK(args[0]->IsString());
+  Local<String> envtag = args[0].As<String>();
+  String::Utf8Value strenvtag(envtag);
+  std::string text;
+  if (!SafeGetenv(*strenvtag, &text)) return;
+  args.GetReturnValue()
+  .Set(String::NewFromUtf8(args.GetIsolate(), text.c_str()));
```

Comment:
Can you indent this line by 4 spaces? We generally do that for statement continuations, see https://github.com/nodejs/node/blob/master/CPP_STYLE_GUIDE.md#4-spaces-of-indentation-for-statement-continuations

Reply:
sure 👍